src/core/src/op/convert.cpp

```diff
@@ -26,12 +26,14 @@ struct Evaluate : public element::NoAction<bool> {
     template <element::Type_t ET, class TI = fundamental_type_for<ET>>
     static result_type visit(const Tensor& arg, Tensor& out, const size_t count) {
         using namespace ov::element;
-        return IfTypeOf<CONVERT_ET_LIST>::apply<EvalByOutputType<is_lp_type(ET)>>(
-            out.get_element_type(),
-            reinterpret_cast<const TI*>(arg.data()),
-            out,
-            count,
-            ET);
+        return IF_TYPE_OF(Convert_out,
+                          CONVERT_ET_LIST,
```

Comment:
Is it expected that `CONVERT_ET_LIST` not wrapped with `OV_PP_ET_LIST`. Could you explain in detail how `OV_PP_ET_LIST` works?